src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,24 @@
-export const PostInfo = () => <>Put the post here</>;
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <UserInfo user={post.user} />
+    </div>
+
+    <p className="PostInfo__body">{post.body}</p>
+
+    {post.comments.length > 0 ? (
```

Comment:
```suggestion
    {!!post.comments.length ? (
```
**PLS DO THE SAME FOR ALL SUCH CASES IN THE PROJECT**